### PR TITLE
Tycho test 1.0

### PR DESCRIPTION
**Pull request by GloriaWa**
Trying to use maven+tycho+Travis CI
build result screenshot attached with this draft pr

### CODE REVIEW

**Comment by GloriaWa** [comment]
build_result

**Comment by GloriaWa** [comment]
I am not sure if this is a proper maven build...for me, it seems like the build logs are too few...hmmm...

File: .classpath [hunk 1]
```diff
@@ -1,11 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-12">
-		<attributes>
-			<attribute name="module" value="true"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
```
Comment (GloriaWa) [inline .classpath:3]: The changes here will not be added to the final PR. It is the environment I am currently working with...should not be submitted to the master branch.

File: .settings/org.eclipse.jdt.core.prefs [hunk 2]
```diff
@@ -1,9 +1,15 @@
 eclipse.preferences.version=1
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=12
-org.eclipse.jdt.core.compiler.compliance=12
```
Comment (GloriaWa) [inline .settings/org.eclipse.jdt.core.prefs:3]: The changes here will not be added to the final PR. It is the environment I am currently working with...should not be submitted to the master branch.

File: META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -15,5 +15,6 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.lsp4j,
  org.eclipse.lsp4j.jsonrpc
 Automatic-Module-Name: LSP_CheckerFramework
-Bundle-RequiredExecutionEnvironment: JavaSE-12
```
Comment (GloriaWa) [inline META-INF/MANIFEST.MF:18]: The changes here will not be added to the final PR. It is the environment I am currently working with...should not be submitted to the master branch.

**Review by wmdietl — CHANGES_REQUESTED** [review]
@GloriaWa Can you please merge in master to resolve the merge conflicts?

**Pull request closed without merging**